test/messaging/adapters/amqp/consumer_test.exs

```diff
@@ -0,0 +1,81 @@
+defmodule Wanda.Messaging.Adapters.AMQP.ConsumerTest do
+  use ExUnit.Case
+
+  import Mox
+
+  alias Trento.Checks.V1.{
+    ExecutionRequested,
+    FactsGathered
+  }
+
+  alias Wanda.Messaging.Adapters.AMQP.Publisher
+
+  setup [:set_mox_from_context, :verify_on_exit!]
+
+  @moduletag :integration
```

Comment:
How does this `:integration` tag work?

Reply:
you can skip integration tests in local with `mix test --exclude integration`